packages/create-project/src/generateProject.ts

```diff
@@ -36,10 +36,14 @@ export async function generateProject(templateId: TemplateId, outputDirectory: s
             args.push("--provider", `"${provider!}"`);
             if (projectName) {args.push("--project-name", `"${projectName!}"`);}
             break;
+        case "typescript-library":
+            commandName = "generate-typescript-library";
+            args.push("--host-scope", `"${hostScope!}"`);
```

Comment:
i'm looking at the template you created, and there is no "HOST_SCOPE" variable used, unless i missed it. Is it really required to ask this question, our should package-name be enough?

Reply:
Package name can be enought, I wasn't sure if in the template PR I will have comment about forgetting to add `HOST_SCOPE`, so I let it in here just in case. I will clean that.